common/tier4_autoware_utils/include/tier4_autoware_utils/geometry/boost_polygon_utils.hpp

```diff
@@ -28,7 +28,6 @@
 
 namespace tier4_autoware_utils
 {
-namespace bg = boost::geometry;
```

Comment:
This may affect downstream packages.
If there are any errors, please fix the planning packages.

Reply:
build-and-test-differential has no error.
Do you have any other items which should be confirmed?

Reply:
Hmm, then it might be okay.